instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+
+class Login extends React.Component {
+    constructor(){
+        super();
+        this.setState = {}
+    }
+    loggedIn = (e) => {
+        e.preventDefault();
+        const user = document.getElementById("user").value;
+        console.log(user.value)
+        localStorage.setItem('user', user)
+        window.location.reload()
+      }
+      render() {
+    return (
```

Comment:
Since we will be attaching this method to an `onSubmit` event, `window.location.reload()` will not be necessary here (as long as we remove `e.preventDefault()` in this method).